internal/nitro-network/main.go

```diff
@@ -79,7 +80,8 @@ func InitializeNitroNetwork() error {
 		dataFolder, cleanup := utils.GenerateTempStoreFolder()
 		defer cleanup()
 
-		server, node, msgService, err := interRpc.InitChainServiceAndRunRpcServer(nodeOpts.Pk, chainOpts, nodeOpts.UseDurableStore, dataFolder, false, nodeOpts.MsgPort, nodeOpts.RpcPort)
+		var emptyFS fs.FS // Serve a dummy static site (we do not currently support embedding the GUI here)
+		server, node, msgService, err := interRpc.InitChainServiceAndRunRpcServer(nodeOpts.Pk, chainOpts, nodeOpts.UseDurableStore, dataFolder, false, nodeOpts.MsgPort, nodeOpts.RpcPort, emptyFS)
```

Comment:
This isn't ideal but difficult to work around. We can't embed anything further up the directory tree than this file.

Options:
*place `packages` underneath `internal`
* our build script needs to copy the static site somewhere where it can be embedded.